src/app/helptext/data-protection/cloudsync/cloudsync-form.ts

```diff
@@ -178,4 +178,6 @@ export default {
 
   cloudsync_task_add: T('Add Cloud Sync Task'),
   cloudsync_task_edit: T('Edit Cloud Sync Task'),
+
+  empty_src_dirs_placeholder: T('Create empty source dirs on destination after sync'),
```

Comment:
Seems like unrelated to this PR

Reply:
That is the code of another PR that already merged.
At the work beginning on this ticket I have pulled from a branch `NAS-117153` that was already merged, so the commits under that were added into this branch.